templates/master/01-master-kubelet/_base/units/kubelet.service.yaml

```diff
@@ -10,8 +10,6 @@ contents: |
   [Service]
   Type=notify
   ExecStartPre=/bin/mkdir --parents /etc/kubernetes/manifests
-  ExecStartPre=/bin/rm -f /var/lib/kubelet/cpu_manager_state
```

Comment:
Some of the platform-specific templates also has this in it still. Those will still run this on top of the cleanup services. We should remove all references there as well if this is for all platforms

Reply:
Oh, you are right, let me check that.

Reply:
Fixed.